src/assets/button-icon-disabled.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These icons are similar. Can we use just one and change the fill in specific conditions?

Reply:
@hayuna  Is that should be possible to change icon by a prop ?

Reply:
It should be. Components should be reusable. What if we want to and another icon in different place? Should we create another component to this purpose? Or just change one props?

Reply:
Im worried there is no "clean" way to do that or I just can't :p

Reply:
@Programmers-Only-Group/frontend-team Could you help him to achieve this?

Reply:
@Krysik https://stackoverflow.com/a/54521330